cmd/fireeth/poller.go

```diff
@@ -73,7 +74,28 @@ func newGenericEVMPollerCmd(logger *zap.Logger, tracer logging.Tracer) *cobra.Co
 	return cmd
 }
 
+func newFirehoseTracerPollerCmd(logger *zap.Logger, tracer logging.Tracer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "firehose-tracer-api <rpc-endpoint> <first-streamable-block>",
+		Short: "poll blocks using debug_traceFirehoseBlockByNumber",
```

Comment:
Add `Description:` explain a bit what this poller is about. Also add an `*Experimental*` note at the end of the description explaining this is not production ready and should not be used outside of development purposes.

You can use:

```
   Description: cli.Dedent(`
     Some long description
     
     Over multiple lines.
   `),
```

Which makes things easier to read in the source.